terraform/main.tf

```
@@ -0,0 +1,34 @@
+module "dns_zone" {
+  source = "./modules/dns"
+
+  domain = "domain.tld"
+  record_sets = [
```

```suggestion
  zone_domain = "domain.tld"
  recordsets = [
```
Les attributs utilisés doivent correspondre à ceux définis dans les variables du module source.